storio-content-resolver/src/androidTest/java/com/pushtorefresh/storio/content_resolver/impl/ObserveChangesTest.java

```diff
@@ -16,15 +21,17 @@
 import rx.functions.Action1;
 import rx.schedulers.Schedulers;
 
-public class ObserveChangesTest extends BaseSubscriptionTest {
+@RunWith(AndroidJUnit4.class)
```

Comment:
@nikitin-da without this annotation your tests in this class didn't run, and I had to ignore them for now, because `ProviderTestCase2` does not call contentObserver :(